packages/zowe-explorer/package.json

```diff
@@ -1913,26 +1913,26 @@
     ]
   },
   "scripts": {
-    "build": "gulp build && yarn madge && yarn license && webpack --mode development",
-    "build:integration": "yarn createTestProfileData && yarn license && tsc --pretty --project tsconfig-tests.json",
+    "build": "gulp build && pnpm madge && pnpm license && webpack --mode development",
+    "build:integration": "pnpm createTestProfileData && pnpm license && tsc --pretty --project tsconfig-tests.json",
     "test:unit": "jest \".*__tests__.*\\.unit\\.test\\.ts\" --coverage",
     "test:theia": "mocha ./out/__tests__/__theia__/*.theia.test.js --timeout 50000 --reporter mocha-multi-reporters --reporter-options configFile=.mocharc.json",
-    "test": "yarn test:unit",
-    "vscode:prepublish": "gulp build && webpack --mode production && yarn updateStrings && yarn run compile-web",
+    "test": "pnpm test:unit",
+    "vscode:prepublish": "gulp build && webpack --mode production && pnpm updateStrings && pnpm run compile-web",
     "updateStrings": "node ../../scripts/stringUpdateScript.js && prettier --write --loglevel warn ./i18n package.nls*",
-    "package": "vsce package --yarn && node ../../scripts/mv-pack.js vscode-extension-for-zowe vsix",
+    "package": "vsce package --no-dependencies --allow-star-activation && node ../../scripts/mv-pack.js vscode-extension-for-zowe vsix",
```

Comment:
The `--allow-star-activation` shouldn't be necessary on the "next" branch because we stopped activating ZE that way in #2319